### PR TITLE
add iceberg properties

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/<add-your-issue-number-here>
Review deadline:

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/reference/pages/properties/cluster-properties.adoc [hunk 1]
```diff
@@ -1534,6 +1578,96 @@ Enables the translation of topic data into Iceberg tables. Setting `iceberg_enab
...
+
+*Type:* string
+
+*Default:* `null`
+
+---
+
+=== iceberg_rest_catalog_token
+
+Token used to access the REST Iceberg catalog. If the token is present, Redpanda ignores credentials stored in the properties iceberg_rest_catalog_user_id and iceberg_rest_catalog_secret.
```
Comment (Deflaimun) [inline modules/reference/pages/properties/cluster-properties.adoc]:
```suggestion
Token used to access the REST Iceberg catalog. If the token is present, Redpanda ignores credentials stored in the properties `iceberg_rest_catalog_user_id` and `iceberg_rest_catalog_secret`.
```

File: modules/reference/pages/properties/cluster-properties.adoc [hunk 2]
```diff
@@ -1534,6 +1578,96 @@ Enables the translation of topic data into Iceberg tables. Setting `iceberg_enab
...
+
+*Type:* string
+
+*Default:* `null`
+
+---
+
+=== iceberg_translation_interval_ms_default
+
+How often an Iceberg enabled topic is checked for new data to translate. You can override this value at topic level using redpanda.iceberg.translation.interval.ms.
```
Comment (Deflaimun) [inline modules/reference/pages/properties/cluster-properties.adoc]:
```suggestion
How often an Iceberg enabled topic is checked for new data to translate. You can override this value at topic level using `redpanda.iceberg.translation.interval.ms`.
```

File: modules/reference/pages/properties/cluster-properties.adoc [hunk 3]
```diff
@@ -1534,6 +1578,96 @@ Enables the translation of topic data into Iceberg tables. Setting `iceberg_enab
...
+
+*Requires restart:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `std::chrono::milliseconds(1min)`
```
Comment (Deflaimun) [inline modules/reference/pages/properties/cluster-properties.adoc]:
```suggestion
*Default:* `60000` (1min)
```

File: modules/reference/pages/properties/cluster-properties.adoc [hunk 4]
```diff
@@ -1520,6 +1520,52 @@ A list of supported HTTP authentication mechanisms.
 
 ---
 
+=== iceberg_catalog_base_location
+
+Base path for the cloud object storage-backed Iceberg catalog. After Iceberg is enabled, do not change this value.
```
Comment (kbatuigas) [inline modules/reference/pages/properties/cluster-properties.adoc]: @Deflaimun can we find out if we mean to say both "cloud" and "object"..? I think @andrwng might be able to help

File: modules/reference/pages/properties/cluster-properties.adoc [hunk 5]
```diff
@@ -1534,6 +1580,96 @@ Enables the translation of topic data into Iceberg tables. Setting `iceberg_enab
 
 ---
 
+=== iceberg_rest_catalog_endpoint
+
+URL of Iceberg REST catalog endpoint.
```
Comment (kbatuigas) [inline modules/reference/pages/properties/cluster-properties.adoc]: @andrwng is this meant to be set together with / as a pair with `iceberg_catalog_base_location`?

File: modules/reference/pages/properties/cluster-properties.adoc (hunk 4 above)
Comment (Deflaimun) [reply modules/reference/pages/properties/cluster-properties.adoc]: @andrwng on docs we usually side with object storage. Can I make this change here  ?

File: modules/reference/pages/properties/cluster-properties.adoc [hunk 6]
```diff
@@ -1522,6 +1540,66 @@ A list of supported HTTP authentication mechanisms.
 
 ---
 
+=== iceberg_catalog_base_location
+
+Base path for the cloud object storage-backed Iceberg catalog. After Iceberg is enabled, do not change this value.
```
Comment (Deflaimun) [inline modules/reference/pages/properties/cluster-properties.adoc]:
```suggestion
Base path for the object storage backed Iceberg catalog. After Iceberg is enabled, do not change this value.
```

**Review by rockwotj — APPROVED** [review]

**Review by kbatuigas — APPROVED** [review]

**Pull request merged**